src/platform_impl/linux/window.rs

```diff
@@ -101,17 +107,20 @@ impl Window {
       .insert(window_id);
 
     // Set Width/Height & Resizable
-    let win_scale_factor = window.scale_factor();
     let (width, height) = attributes
       .inner_size
       .map(|size| size.to_logical::<f64>(win_scale_factor as f64).into())
       .unwrap_or((800, 600));
-    window.set_default_size(1, 1);
-    window.resize(width, height);
+
+    let window_clone = window.clone();
+    glib::idle_add_local_once(move || {
+      window_clone.set_resizable(true);
+      window_clone.resize(width, height);
+    });
 
     if attributes.maximized {
       let maximize_process = util::WindowMaximizeProcess::new(window.clone(), attributes.resizable);
-      glib::idle_add_local_full(glib::Priority::HIGH_IDLE, move || {
+      glib::idle_add_local_full(glib::Priority::DEFAULT_IDLE, move || {
```

Comment:
A dispatching error occurred when maximize was set to HIGH_IDLE, so I changed it to DEFAULT_IDLE. The error occurred even when wrapping the header, resize and maximize in HIGH_IDLE using idle_add, so I changed only maximize to DEFAULT_IDLE.

<img width="927" alt="스크린샷 2024-10-02 오후 6 17 35" src="https://github.com/user-attachments/assets/8f262963-bb53-46d0-a878-8f01e0d7c0d2">